### ISSUE TITLE
how to align components in multiple line as arguments?

### ISSUE THREAD

**Issue by wangsongiam** [yannickcr/eslint-plugin-react#1584]
Hi, could anyone help me with this case?  I've tried a couple of rules but couldn't find out how to change
```js
  replaceBodyHTMLString(renderToString(<Provider store={store}>
    <StyleSheetManager sheet={sheet.instance}>
      {bodyComponent}
    </StyleSheetManager>
  </Provider>))
```
to this
```js
  replaceBodyHTMLString(
    renderToString(
      <Provider store={store}>
        <StyleSheetManager sheet={sheet.instance}>
          {bodyComponent}
        </StyleSheetManager>
      </Provider>
    )
  )
```
eslintrc
```json
{
  "env": {
    "browser": true,
    "es6": true,
    "node": true
  },
  "extends": "eslint:recommended",
  "parserOptions": {
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true,
      "jsx": true
    },
    "sourceType": "module"
  },
  "plugins": ["react"],
  "rules": {
    "indent": [2, 2],
    "linebreak-style": [2, "unix"],
    "quotes": [2, "double", { "allowTemplateLiterals": true }],
    "semi": [2, "never"],
    "brace-style": [2, "1tbs", { "allowSingleLine": true }],
    "curly": [2, "all"],
    "no-console": [2, { "allow": ["warn", "error", "log"] }],
    "no-unused-vars": 0,
    "function-paren-newline": ["error", "multiline"],
    "comma-dangle": [2, "only-multiline"]
  }
}

```

**Comment by ljharb** [comment]
That isn't a jsx concern nor a react one; you'd use an eslint rule to govern that - but I don't think one exists (or if it does, I'm not sure which it'd be).

Either way, the `jsx-indent` rule would help once the jsx was moved onto its own line.

**Comment by wangsongiam** [comment]
@ljharb Thanks for the help, `jsx-indent` definitely fills part of the job, I'll dig more with that and `eslint`

**Comment by wangsongiam** [comment]
@jomasti Thanks for the close lookup, that was me frustratingly trying out splitting multi-arguments for the case I am asking, did not entirely understand the compatibilities. 🤐

**Comment by wangsongiam** [comment]
Ok, I found the solution, and FWIW, this is the final eslintrc.json
```json
{
  "env": {
    "browser": true,
    "es6": true,
    "node": true
  },
  "extends": "eslint:recommended",
  "parserOptions": {
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true,
      "jsx": true
    },
    "sourceType": "module"
  },
  "plugins": ["react"],
  "rules": {
    "indent": [2, 2],
    "linebreak-style": [2, "unix"],
    "quotes": [2, "double", { "allowTemplateLiterals": true }],
    "semi": [2, "never"],
    "brace-style": [2, "1tbs", { "allowSingleLine": true }],
    "curly": [2, "all"],
    "no-console": [2, { "allow": ["warn", "error", "log"] }],
    "no-unused-vars": 0,
    "comma-dangle": [2, "only-multiline"],
    "react/jsx-indent": [2, 2]
  }
}
```
Note that it won't trigged warning or error if having the format like I did from the original issue.
